platform/ui/src/components/userPreferencesForm/UserPreferencesForm.styl

```diff
@@ -15,3 +15,14 @@
 .ModalHeader
```

Comment:
Are we still using this style? Do we need to remove it to make sure it doesn't accidentally impact something else?

Reply:
This was fixed with the latest changes. thx.

Reply:
This was fixed with the latest changes. thx.